apps/frontend/src/pages/project/project.tsx

```diff
@@ -0,0 +1,63 @@
+import { type ProjectFindRequestDto } from "@git-fit/shared";
+import { useParams } from "react-router-dom";
+
+import { Loader, PageLayout } from "~/libs/components/components.js";
+import { DataStatus } from "~/libs/enums/enums.js";
+import {
+	useAppDispatch,
+	useAppSelector,
+	useEffect,
+} from "~/libs/hooks/hooks.js";
+import { actions as projectActions } from "~/modules/projects/projects.js";
+import { NotFound } from "~/pages/not-found/not-found.jsx";
+
+import styles from "./styles.module.css";
+
+const Project = (): JSX.Element => {
+	const dispatch = useAppDispatch();
+	const { id: projectId } = useParams<ProjectFindRequestDto>();
```

Comment:
let's define type for useParams inline
```suggestion
	const { id: projectId } = useParams<{ id: string }>();
```